README.md

```diff
@@ -97,24 +97,14 @@ process.mainArgs = process.argv.slice(process._exec ? 1 : 2)
 const { parseArgs } = require('@pkgjs/parseargs');
 ```
 
-```js
-// unconfigured
-const { parseArgs } = require('@pkgjs/parseargs');
-const args = ['-f', '--foo=a', '--bar', 'b'];
-const options = {};
-const { values, positionals } = parseArgs({ args, options });
-// values = { f: true, foo: 'a', bar: true }
-// positionals = ['b']
-```
-
 ```js
 const { parseArgs } = require('@pkgjs/parseargs');
 // type:string
```

Comment:
```suggestion
// type:string and type:boolean
```

Reply:
This example is a bit clumsy now with `f` declared as a slightly funky long/short to cover the previous implicit short. Making a series of suggestions to refactor a little.

Reply:
Going to be too messy, I'll make a single comment with code in an ordinary comment!